spec/Iterators.tex

```diff
@@ -37,8 +47,15 @@ \section{Iterator Definitions}
 \item \chpl{yield} statements may appear in the body of an iterator, but not in
 a procedure.
 \item A \chpl{return} statement in the body of an iterator is not allowed to have an expression.
+\item The intent and type specified after the argument list refer to the type
+yielded, not the type returned (see previous bullet).  However, they are
+syntactically the same as a \sntx{return-intent} and a \sntx{return-type}.
 \end{itemize}
 
+\begin{openissue}
```

Comment:
Should this be a `craychapel` note? I think that's used for differences between the spec and implementation and openissue is used more for things that aren't resolved within the spec itself?

Reply:
Oh, I guess it's also an `openissue` in `Procedures.tex`. I guess I don't have a strong opinion then